core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
Also regarding the debugging I would just use `debug=true` (and not debug_instruction=true and debug_details=true etc) and then show **all** path details. If this is too much in the UI one can always pick just a few or one path detail or we can later think to improve this.